packages/analytics-js-integrations/__tests__/integrations/GoogleAds/browser.test.js

```diff
@@ -76,6 +98,24 @@ describe('Scenario to test dynamic remarketing event by enabling the dynamicRema
 });
 
 // New Config Test Cases
+describe('Edge scenarios', () => {
+  let googleAds;
+  beforeEach(() => {
+    googleAds = new GoogleAds(googleAdsConfigs[2], {  }, destinationInfo);
+    googleAds.init();
+    window.gtag = jest.fn();
+  });
+  test('Scenario to test no event name', () => {
```

Comment:
Can you add a test with page call also? We are using same function in page calls as well.

Reply:
addressed